### DISCUSSION TITLE
Complience with deck.gl v9

### DISCUSSION THREAD

**Discussion by vaduga** [uber/nebula.gl#893] [General]
"import {Deck} from '@deck.gl/core/typed'  " - is deprecated in deck.gl v.9 , types exposed at package roots as stated in this PR
https://github.com/visgl/deck.gl/pull/8293

When nebula.gl will be compatible with the new deck.gl versions?
I also encounter lagging (dropped frames) of editablegeojsonlayer in react 18 while panning. It doesn't lag if pickable=false.

**Comment by ibgreen** [comment]
The deck.gl v9 versions need to release first, work is still ongoing. Once that is done, hopefully in Q1 2024, we will need a contributor to step up and make a pass on nebula.

I suppose someone could also make an alpha release of nebula.gl that works on 9.0-alpha releases. We'd welcome such a contribution, though it may require a bit more work.

**Reply by ffendt** [reply]
Now that deck.gl v9 is released, is there anyone working on this yet? Can you tell, if this will be a big effort?

**Reply by vaduga** [reply]
it's all working with deck v9 . Many thanks to the contributors!
layers are available as part of deck.lg-community package now https://github.com/visgl/deck.gl-community/tree/master/modules

**Reply by ffendt** [reply]
Wow nice, thanks 👍

**Comment by AliAlawi96** [comment]
is there a version of nebula.gl package that I can import that works with deck v9 ?

**Reply by ibgreen** [reply]
> is there a version of nebula.gl package that I can import that works with deck v9 ?

No, but see https://github.com/uber/nebula.gl/discussions/901
